src/test/update-status.test.js

```diff
@@ -0,0 +1,26 @@
+import Todo from '../TodoClass.js';
+import createTodoItem from '../CreateTodoItem.js';
+
+describe('Make test for status of completed ', () => {
+  test('Update todo items status', () => {
+    const todo = new Todo('Make tests for this!!');
+    todo.changeStatus(true);
+    expect(todo.isCompleted).toBe(true);
+    // const todoNode = createTodoItem(todo);
```

Comment:
- [ ] Removing commented code not only makes it harder to find later, but it also makes it so people in the future don't know it existed before. For finding the code, there are [tools and git commands](http://stackoverflow.com/q/278192/971592) to help you look at the history of a file. Please remove all commented code and unnecessary files from your code base :smile: